_layouts/about.html

```diff
@@ -4,30 +4,28 @@
 
 <div class="about-page">
   <head>
-      <title>{{ site.author }}</title>
+    <title>{{ site.author }}</title>
   </head>
   <div class="post-back">
     <a class="black-link" href="{{ site.url | relative_url }}"> ← {{ site.moving.back_to }} </a>
   </div>
-
-  {%- if site.moving.avatar_url -%}
-    <img class="about-avatar" src="{{ site.moving.avatar_url }}"/>
-  {%- endif -%}
-
-  {%- if site.author -%}
-    <div class="about-your-name">
-    {{ site.author | escape }}
-    </div>
-  {%- endif -%}
-
-  {%- if site.moving.about_you -%}
-    <p class="about-info">
+  <div class="header">
```

Comment:
Please don’t format code like this.

Reply:
I don't quite understand what you're saying. Could you show me an example?

Reply:
> Please don’t format code like this.

I don't quite understand what you're saying. Could you show me an example?

Reply:
Just follow the original.

Reply:
Anyway, let me do this code format.